Titan/Managers/ThreadManager.cs

```diff
@@ -84,22 +69,44 @@ public void StartReport(TitanAccount account, ReportInfo info)
                         case Result.NoMatches:
                             _log.Error("Could not receive match information for {Account}: User is not in live match.",
                                        account._liveGameInfo.SteamID.ConvertToUInt64());
+                            _failCount++;
                             break;
                         case Result.TimedOut:
                             _log.Error("Processing thread for {Account} has timed out.");
+                            _failCount++;
                             break;
                         case Result.SentryRequired:
                             _log.Error("The account has 2FA enabled. Please set {sentry} to {true} " +
                                        "in the accounts.json file.", "sentry", true);
+                            _failCount++;
                             break;
                         case Result.RateLimit:
                             _log.Error("The Steam Rate Limit has been reached. Please try again in a " +
                                        "few minutes.");
+                            _failCount++;
                             break;
                         case Result.Code2FAWrong:
                             _log.Error("The provided SteamGuard code was wrong. Please retry.");
+                            _failCount++;
                             break;
                     }
+
+                    if (_taskDic.Count - _count - _failCount == 0)
+                    {
+                        _log.Information("SUCCESS! Titan has successfully sent {Amount} out of {Total} reports to {Target}.",
+                            _count, _taskDic.Count, account._reportInfo.SteamID.ConvertToUInt64());
+
+                        Titan.Instance.UIManager.SendNotification(
+                            "Titan", _count + " reports have been successfully sent!"
+                        );
+
+                        account.IsLast = false;
```

Comment:
Now that this is a different way to implement this, the `IsLast` variable can be removed from TitanAccount. Please do this.